### PR TITLE
[docs] Polish to match standard

### PR DESCRIPTION
**Pull request by oliviertassinari**
I noticed this from #4306 that fix the broken image reported in https://app.ahrefs.com/site-audit/3992793/data-explorer?columns=pageRating%2Curl%2ChttpCode%2CcontentType%2CincomingImage&current=01-11-2024T025516&filterId=c7bd3d737800c382ec57c0adace75595&issueId=c0e87084-feaf-11e8-8c05-001e67ed4657&sorting=-pageRating (great to see this fixed beforehand!)

### CODE REVIEW

File: docs/data/toolpad/studio/getting-started/installation.md [hunk 1]
```diff
@@ -94,14 +94,14 @@ Now you can start your Toolpad Studio application using one of the commands:
 npm run toolpad-studio:dev
 ```
 
-```bash yarn
```
Comment (oliviertassinari) [inline docs/data/toolpad/studio/getting-started/installation.md:97]: Iterate on https://github.com/mui/mui-public/issues/181.

File: docs/data/toolpad/core/introduction/installation.md [hunk 2]
```diff
@@ -4,47 +4,47 @@ title: Toolpad Core - Installation
...
 
-</codeblock>
+```bash yarn
+yarn add @mui/material @mui/icons-material @emotion/react @emotion/styled
+```
 
-## Automatic Installation
+</codeblock>
 
-<p class="description">Learn how to install Toolpad Core in your local environment.</p>
```
Comment (oliviertassinari) [inline docs/data/toolpad/core/introduction/installation.md:47]: Wrong position

File: docs/data/toolpad/core/introduction/installation.md [hunk 3]
```diff
@@ -4,47 +4,47 @@ title: Toolpad Core - Installation
...
+```
+
 </codeblock>
 
 The Toolpad Core package has a peer dependency on `@mui/material` and `@mui/icons-material`. If you aren't using these already in your project, you can install them with:
 
 <codeblock storageKey="package-manager">
 
 ```bash npm
-npm install -S @mui/material @mui/icons-material @emotion/react @emotion/styled
```
Comment (oliviertassinari) [inline docs/data/toolpad/core/introduction/installation.md:32]: Remove the `-S` isn't this the default https://docs.npmjs.com/cli/v10/commands/npm-install#save?

File: docs/data/toolpad/core/introduction/installation.md [hunk 4]
```diff
@@ -4,47 +4,47 @@ title: Toolpad Core - Installation
 
 # Installation
 
-## Manual Installation
+<p class="description">Learn how to install Toolpad Core in your local environment.</p>
+
+## Manual installation
```
Comment (oliviertassinari) [inline docs/data/toolpad/core/introduction/installation.md:9]: Sentence case

Comment (Janpot) [reply docs/data/toolpad/core/introduction/installation.md:9]: Can we make vale catch these?

File: docs/data/toolpad/core/components/page-container/page-container.md [hunk 5]
```diff
@@ -47,7 +47,7 @@ The breadcrumbs contains **ACME / Home / Orders** when you visit the path **/hom
 
 {{"demo": "TitleBreadcrumbsPageContainer.js", "height": 300, "hideToolbar": true}}
 
-## Dynamic Routes
+## Dynamic routes
```
Comment (oliviertassinari) [inline docs/data/toolpad/core/components/page-container/page-container.md:50]: @bharatkashyap Similar to #4306, I fixed a couple more to match https://www.notion.so/mui-org/Writing-style-guide-2a957a4168a54d47b14bae026d06a24b.

cc @samuelsycamore we are progressively getting to match the style guide everywhere 😄

File: docs/data/toolpad/core/introduction/installation.md (hunk 4 above)
Comment (oliviertassinari) [reply docs/data/toolpad/core/introduction/installation.md:9]: We could use https://vale.sh/docs/topics/styles/#capitalization. Now,

- There are valid uses of title cases when we reference components, e.g. https://mui.com/toolpad/core/react-app-provider/#next-js-app-router seems right, it's how Next.js labels it. 
- Vale CI is broken: #4347.

File: docs/data/toolpad/studio/index.md [hunk 6]
```diff
@@ -4,6 +4,6 @@ title: Toolpad Studio docs - Home
 
 # Toolpad Studio
 
-<p class="description">Toolpad Studio docs</p>
+<p class="description">Toolpad Studio docs.</p>
```
Comment (oliviertassinari) [inline docs/data/toolpad/studio/index.md:7]: We can enforce this in docs-infra: https://github.com/mui/material-ui/pull/44292.

**Pull request merged**